memcached/internal/memcached.c

```diff
@@ -51,6 +51,20 @@
 __thread struct slab_cache slabc;
 struct quota quota;
 struct slab_arena arena;
+struct slab_arena_info slab_info;
+
+struct slab_arena_info *
+memcached_slab_arena_info()
+{
+	slab_info.arena_size = arena.used;
+
+	slab_info.quota_used = quota_used(arena.quota);
+	slab_info.quota_size = quota_total(arena.quota);
+	slab_info.quota_used_ratio = 100 * ((double) slab_info.quota_used /
+					((double) slab_info.quota_size + 0.0001));
+
+	return &slab_info;
+}
```

Comment:
`arena_used` and `arena_used_ratio` are not filled here. And, it seems, they are never filled and always zero in current implementation.

Reply:
Yes, it is so. I mentioned it in commit message:

>    With our own slab arena we need a way to obtain information about arena.
>    It's a reason why new module function slab() was introduced. Output of
>    slab() is similar to output from `box.slab.info()`, but without
>    items_size, items_used, items_used_ratio and arena_used is not
>    implemented.

The initial idea was to make the same info as in `box.slab.info()`, but then I decided to don't implement `arena_used` because don't know how to track its usage.

Reply:
So we should not report zeros here.

Reply:
ok, I'll remove not implemented counters.

Reply:
removed